.envrc

```diff
@@ -0,0 +1,5 @@
+# Have aws-vault use the login keychain for better timeout behavior
+export AWS_VAULT_KEYCHAIN_NAME=login
+
+# Uses Go Module
+export GO111MODULE=auto
```

Comment:
Per https://trussworks.slack.com/archives/CBRSZT93N/p1571187661082100 setting 
`GO111MODULE=auto` is no longer needed in 1.13. We can remove it here and from CircleCI.

Reply:
I removed here. Is this what you mean re: CircleCI?
https://github.com/trussworks/truss-aws-tools/pull/73/files#diff-1d37e48f9ceff6d8030570cd36286a61L8